third_party/amd/lib/TritonAMDGPUTransforms/BlockPingpong.cpp

```diff
@@ -162,8 +162,13 @@ LogicalResult Pingponger::genLocalSlice(OpBuilder &builder, Value v,
     return failure();
   auto dotOperandEnc = ttg::DotOperandEncodingAttr::get(
       builder.getContext(), opIdx, dotEncoding, kWidth);
+  SmallVector<int64_t> allocShape = llvm::to_vector(type.getAllocShape());
```

Comment:
Is it expanding the shape to 3d? If so, seems fine to me?

Reply:
Actually it's not expanding and I just copied it. Fixed it to just use the original shape